scripts/Muon/GUI/Common/dummy/dummy_widget.py

```diff
@@ -1,8 +1,8 @@
 from __future__ import (absolute_import, division, print_function)
 
 
-from Muon.GUI.dummy.dummy_view import DummyView
-from Muon.GUI.dummy.dummy_presenter import DummyPresenter
+from Muon.GUI.Common.dummy.dummy_view import DummyView
+from Muon.GUI.Common.dummy.dummy_presenter import DummyPresenter
 
 
 class DummyWidget(object):
```

Comment:
GitHub won't let me comment inline below here but there is a typo on line 17 in `DummyWidget.` It reads `presneter` and not `presenter`